microcosm_flask/templates/swagger_ui.py

```diff
@@ -0,0 +1,42 @@
+# flake8: noqa
+html = """
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
+    <!-- <link rel="stylesheet" type="text/css" href="./swagger-ui.css" /> -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/swagger-ui.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/index.min.css">
+    <link rel="icon" type="image/png" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/favicon-16x16.png" sizes="16x16" />
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+    <script src="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/swagger-ui-bundle.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/swagger-ui-standalone-preset.min.js"></script>
```

Comment:
We really need to host these locally (otherwise we're giving a third party the ability to run arbitrary scripts on the browser of a person who has access to the API).

Reply:
🤔 I wonder if it could just go in here: https://github.com/globality-corp/globality-static

Reply:
Seems like a good place 👍 Added:
https://github.com/globality-corp/globality-static/pull/24